website/docs/gen-ai/inference/nvidia-nim-llama3.md

```diff
@@ -0,0 +1,345 @@
+---
+title: NVIDIA NIM Llama-3 Deployment on Amazon EKS
+sidebar_position: 8
+---
+import CollapsibleContent from '../../../src/components/CollapsibleContent';
+
+:::danger
+
+Note: Use of this Llama-3 Instruct model is governed by the Meta license.
+In order to download the model weights and tokenizer, please visit the [website](https://huggingface.co/meta-llama/Meta-Llama-3-8B) and accept the license before requesting access.
+
+:::
+
+:::info
+
+We are actively enhancing this blueprint to incorporate improvements in observability, logging, and scalability aspects.
+:::
+
+# NVIDIA NIM LLM Deployment on Amazon EKS
+
+## What is NVIDIA NIM?
+
+NVIDIA NIM makes it easy for IT and DevOps teams to self-host large language models (LLMs) in their own managed environments while still providing developers with industry standard APIs that allow them to build powerful copilots, chatbots, and AI assistants that can transform their business. Leveraging NVIDIA’s cutting-edge GPU acceleration and scalable deployment, NIM offers the fastest path to inference with unparalleled performance.
+
```

Comment:
Add a note or info section here to mention or link to the NIM pricing or licensing information.

Reply:
sure